modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationOutput.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2014-2018 Andrew Gaul <andrew@gaul.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gaul.annotation_processsor;
+
+import java.io.File;
+
+public final class ModernizerAnnotationOutput {
+
+    public static final String IGNORE_CLASSES_FILE_NAME =
+        "ignore-annotated-classes.txt";
+
+    private ModernizerAnnotationOutput() { }
+
+    public static File getOutputPath(File classOutputDir) {
+        if (classOutputDir.getPath().endsWith("/target/classes") ||
```

Comment:
Can you do these on `getAbsolutePath`?

Reply:
What would be the advantage of using it here? When do we use `getPath` and when do we use `getAbsolutePath` ?

Reply:
I believe that `getPath()` can be relative, meaning it won't start at `/Users/...`. `getAbsolutePath` will return the full absolute path and basically always works, so I just always use it to avoid thinking about the behavior of the other stuff.